examples/Browser/Server/appsettings.json

```diff
@@ -7,7 +7,7 @@
   "AllowedHosts": "*",
   "Kestrel": {
     "EndpointDefaults": {
-      "Protocols": "Http2"
+      "Protocols": "Http1AndHttp2"
```

Comment:
I think this is the default. You can probably remove the "Kestrel" settings altogether.

Reply:
Sure